### PR TITLE
Add `PRIsVALUE`

### PR DESCRIPTION
**Pull request by wagenet**

### CODE REVIEW

**Review by chancancode — COMMENTED** [review]
I am pretty sure this does not work (could use tests). But with this, `RS_PRIsVALUE` would now be nil-terminated, which would be wrong when you try to build the string at runtime.

**Comment by wagenet** [comment]
Hmm... I meant to add tests for this.

**Comment by wagenet** [comment]
@chancancode I rebuilt this on top of #57 (`rb_raise`) since that's where it is used in Helix. It works fine, AFAICT.

File: src/ruby.rs [hunk 1]
```diff
@@ -1450,4 +1487,38 @@ tests! {
...
+
+    #[test]
+    fn test_raise(assert: &mut Assertions) {
+        extern "C" fn __test_raise__(_self: VALUE) -> VALUE {
+            unsafe {
+                let prisvalue =  CStr::from_ptr(PRIsVALUE).to_string_lossy();
+                let format = format!("error: %{} (%+{}) failed", prisvalue, prisvalue);
+                let foo = "foo".to_ruby();
+                rb_raise(rb_eRuntimeError, CString::new(format).unwrap().as_ptr(), foo, foo);
+            }
```
Comment (chancancode) [inline src/ruby.rs]: this is no good, see 
https://github.com/tildeio/libcruby-sys/pull/57#discussion_r217492950 (the comment in the code example)

Comment (wagenet) [reply src/ruby.rs]: Yeah, I agree that it's problematic. Basically just did this for now to show that `PRIsVALUE` works. Probably need to expose a Ruby `sprintf` function or something.

**Comment by wagenet** [comment]
We only use this for `rb_raise` in Helix, though I suspect it could have use elsewhere. I'll leave the PR open, but not need to prioritize it right now.

**Pull request closed without merging**